### PR TITLE
Prefix monitoring identifiers

### PR DESCRIPTION
**Pull request by bion**
Use the app prefix for monitoring stack identifiers to reduce naming collisions.

### CODE REVIEW

File: cloud/aws/templates/aws_oidc/monitoring.tf [hunk 1]
```diff
@@ -1,11 +1,10 @@
-data "aws_caller_identity" "current" {}
-
 resource "aws_prometheus_workspace" "metrics" {
-  alias = "CiviForm_metrics"
+  alias = "${var.app_prefix}-CiviForm_metrics"
 }
 
 resource "aws_grafana_workspace" "CiviForm_metrics" {
-  name                     = "CiviFormMetrics"
+  name                     = "${var.app_prefix}-CiviFormMetrics"
```
Comment (nbeloglazov) [inline cloud/aws/templates/aws_oidc/monitoring.tf:6]: For consistency maybe {var.app_previx}-civiform-metrics unless there is a reason why we need to use different casing here.

File: cloud/aws/templates/aws_oidc/monitoring.tf [hunk 2]
```diff
@@ -14,7 +13,7 @@ resource "aws_grafana_workspace" "CiviForm_metrics" {
 }
 
 resource "aws_iam_role" "grafana_assume_role" {
-  name = "grafana-assume-role"
+  name = "${var.app_prefix}-grafana-assume-role"
   assume_role_policy = jsonencode({
     Version = "2012-10-17"
     Statement = [
```
Comment (nbeloglazov) [inline cloud/aws/templates/aws_oidc/monitoring.tf:19]: Rename civiform_monitoring_role_policy below as well?

File: cloud/aws/templates/aws_oidc/monitoring.tf (hunk 1 above)
Comment (bion) [reply cloud/aws/templates/aws_oidc/monitoring.tf:6]: done

File: cloud/aws/templates/aws_oidc/monitoring.tf (hunk 2 above)
Comment (bion) [reply cloud/aws/templates/aws_oidc/monitoring.tf:19]: done

**Review by nbeloglazov — APPROVED** [review]

**Pull request merged**